Pages/Index.cshtml.cs

```diff
@@ -17,30 +17,24 @@ public class IndexModel : PageModel
     {
         public string GetData(string endpoint)
         {
-            string downloadedData = "";
             using (WebClient webClient = new WebClient())
             {
-                downloadedData = webClient.DownloadString(endpoint);
+                return webClient.DownloadString(endpoint);
             }
-            return downloadedData;
         }
 
         public void OnGet()
         {
-            using (var WebClient = new WebClient())
-            {
                 
-                string jsonstring = GetData("https://data.cityofchicago.org/resource/xq83-jr8c.json");
- //               string jsonstring = WebClient.DownloadString("https://data.cityofchicago.org/resource/xq83-jr8c.json");
-                QuickType.Energy[] energies = QuickType.Energy.FromJson(jsonstring);
+                string jsonString = GetData("https://data.cityofchicago.org/resource/xq83-jr8c.json");
+                QuickType.Energy[] energies = QuickType.Energy.FromJson(jsonString);
 
-                jsonstring = GetData("https://data.cityofchicago.org/resource/tfm3-3j95.json");
-//                jsonstring = WebClient.DownloadString("https://data.cityofchicago.org/resource/tfm3-3j95.json");
-                QuickTypeVehicle.Vehicle[] vehicles = QuickTypeVehicle.Vehicle.FromJson(jsonstring);
+                jsonString = GetData("https://data.cityofchicago.org/resource/tfm3-3j95.json");
+                QuickTypeVehicle.Vehicle[] vehicles = QuickTypeVehicle.Vehicle.FromJson(jsonString);
 
                 List<Affluence> affluence = new List<Affluence>();
 
-                var energy_query = from energy in energies
+                var energyQuery = from energy in energies
```

Comment:
Implemented camel case naming convention for variables

Reply:
Yes... camelCase is better.  We only use _ in variable names when we declare a constant.